nailgun/entities.py

```diff
@@ -6915,3 +6915,95 @@ def update(self, fields=None):
         """
         self.update_json(fields)
         return self.read()
+
+
+class VirtWhoConfig(
+        Entity,
+        EntityCreateMixin,
+        EntityDeleteMixin,
+        EntityReadMixin,
+        EntitySearchMixin,
+        EntityUpdateMixin):
+    """A representation of a VirtWho Config entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'blacklist': entity_fields.StringField(),
+            'debug': entity_fields.BooleanField(),
+            'filtering_mode': entity_fields.IntegerField(
+                choices=[0, 1, 2], default=0, required=True),
+            'hypervisor_id': entity_fields.StringField(
+                choices=['hostname', 'uuid', 'hwuuid'],
+                default='hostname', required=True),
+            'hypervisor_password': entity_fields.StringField(),
+            'hypervisor_server': entity_fields.StringField(required=True),
+            'hypervisor_type': entity_fields.StringField(
+                choices=['esx', 'rhevm', 'hyperv', 'xen', 'libvirt'],
+                default='libvirt', required=True),
+            'hypervisor_username': entity_fields.StringField(required=True),
+            'interval': entity_fields.IntegerField(
+                choices=[60, 120, 240, 480, 720], default=120, required=True),
+            'name': entity_fields.StringField(required=True),
+            'no_proxy': entity_fields.StringField(),
+            'organization_id': entity_fields.IntegerField(required=True),
```

Comment:
Do we need this and `'organization': entity_fields.OneToOneField(Organization)` ?

Reply:
Yes. I believe the difference here is that this `organization_id` is to indicate which Org virt-who guest will be added to. Whereas the other on is for ownership of the object. Both of them are listed in the API doc.

Reply:
I suspect that isn't going to work as expected, I haven't got time right now, but have you tested this with a different `organization` and `organization_id`? I'm assuming one will override the other during `create`.

Reply:
I actually have a question out to the developer about the exact reason why their is two. I would like to get this merged ASAP though, so I can file an issue to come back to this if thats good with you.

Reply:
Not really, I'd prefer not to merge known bugs. Others can obviously override this and ACK this PR